lib/kafka_batcher/collector.ex

```diff
@@ -69,6 +70,15 @@ defmodule KafkaBatcher.Collector do
       @impl KafkaBatcher.Behaviours.Collector
       def add_events(events) do
         GenServer.call(__MODULE__, {:add_events, events})
+      catch
+        _, reason ->
+          Logger.warning(
+            "KafkaBatcher: Couldn't get through to collector",
+            reason: inspect(reason),
+            collector: __MODULE__
+          )
+
+          {:error, :kafka_unavailible}
```

Comment:
What's the policy regarding `{:error, :kafka_unavailible}` error?
Should the library user handle the error in this case?